app/src/db/migrations/20200603145742_011-agriseafoodopscreening-data.js

```diff
@@ -0,0 +1,206 @@
+    })
+    .then(formVersionIds => {
+      const versionCodes = statusCodes.map(c => { return {code: c.code, formVersionId: formVersionIds[0]}; });
+      return knex(`${PREFIX}_version_status_code`).insert(versionCodes);
+    })
+    .then(() => {
+      const data = {
+        name: 'submissionEmail',
+        enabled: true,
+        config: {
+          template: 'confirmation-number-email.html',
+          from: EMAIL_FROM,
+          subject: `${FORM_NAME} Form Submission`,
+          title: `${FORM_NAME} Form Submission`,
+          priority: 'normal',
+          messageLinkText: 'Click to view your Submission',
+          messageLinkUrl: `https://comfort-dev.pathfinder.gov.bc.ca/app/${SLUG}/review`
+        },
+        createdBy: 'migration-006'
+      };
+      return knex(`${PREFIX}_settings`).insert(data);
+    })
+    .then(() => {
+      const data = {
+        name: 'confirmationEmail',
+        enabled: true,
+        config: {
+          template: 'confirmation-number-email.html',
+          from: EMAIL_FROM,
+          to: 'NR.CommonServiceShowcase@gov.bc.ca',
+          subject: `${FORM_NAME} Form Accepted`,
+          title: `${FORM_NAME} Form Accepted`,
+          priority: 'normal',
+          messageLinkText: `Please login to view the details of this ${FORM_NAME} Attestation`,
+          messageLinkUrl: `https://comfort-dev.pathfinder.gov.bc.ca/app/${SLUG}/admin/submission`
+        },
+        createdBy: CREATED_BY
+      };
+      return knex(`${PREFIX}_settings`).insert(data);
+    })
+    .then(() => {
+      const data = {
+        name: 'statusAssignmentEmail',
+        enabled: true,
+        config: {
+          template: 'status-assignment-email.html',
+          from: EMAIL_FROM,
+          subject: `${FORM_NAME} Assignment`,
+          title: `${FORM_NAME} Assignment`,
+          priority: 'normal',
+          message: 'The status of the above attestation has changed. You have been assigned to review this attestation.',
+          messageLinkText: `Please login to view the details of this ${FORM_NAME} Attestation`,
+          messageLinkUrl: `https://comfort-dev.pathfinder.gov.bc.ca/app/${SLUG}/admin/submission`
+        },
+        createdBy: CREATED_BY
+      };
+      return knex(`${PREFIX}_settings`).insert(data);
+    })
+    .then(() => {
+      const data = {
+        name: 'generateSubmissionPdf',
+        enabled: true,
+        config: {
+          template: 'generate-submission-pdf-001.html',
+          templateJson: 'generate-submission-pdf-001.json'
+        },
+        createdBy: CREATED_BY
+      };
+      return knex(`${PREFIX}_settings`).insert(data);
+    })
+    .then(() => {
+      const data = {
+        name: 'accessRequestedEmail',
+        enabled: true,
+        config: {
+          template: 'access-requested-email.html',
+          from: EMAIL_FROM,
+          to: 'NR.CommonServiceShowcase@gov.bc.ca',
```

Comment:
@matthewhall78 i'm guessing we don't know the agri/aqua-culture business area's group email address for receiving notifications yet

Reply:
To should default to NR.CommonServiceShowcase@gov.bc.ca. We set it in the higher environments to the appropriate email address (once we know it...)